scania-digital-design-system/sdds
# feat(modal): align show/hide trigger

**Describe pull-request**  
This PR aligns how the modal is opened/closed with the way it works for the popover. The user either handles the logic all on their own via the `open` prop. Or the pass in either a reference to a "trigger"-element or the pass is a css selector to an element that should open the modal on click.


**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1022

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Modal -> Web Component
3. Test the story to make sure the modal opens/closes correctly.
4. Check notes for the new events and methods.

**Suggested test steps**
